src/cltk/dependency/spacy_dep.py

```diff
@@ -31,34 +27,42 @@ class SpacyWrapper:
 
     def __init__(self,
                  language: str,
+                 spacy_language: spacy.Language = None,
                  interactive: bool = True,
                  silent: bool = False):
         """
 
-        :param language:
-        :param interactive:
-        :param silent:
+        :param language: One of the officially supported languages by CLTK. It selects a default model for
+        :param spacy_language: A `Language` instance from spaCy (not used by default).
+        :param interactive: Are downloads interactive?
+        :param silent: Is it verbose?
         """
-        self.language = language
-        self._interactive = interactive
-        self._silent = silent
+        self.spacy_language = spacy_language
+        if self.spacy_language:
+            self.language = self.spacy_language.lang
+            self.nlps[self.language] = self
+        else:
+
+            self.language = language
+            self._interactive = interactive
```

Comment:
What's the point of this? Do you want the users to not change `interactive` after they have created the object or why do you assign it to `self._interactive` instead of just `self.interactive`.
And if you do it with these two, then why not `language`? Is it fine for the user to change the language after initialization, but not `interactive`?

Reply:
The point is to access useful attributes like `language` and let useless ones like `interactive` unreachable.
The underscore prefix may prevent the developer to reach an attribute, even if it's still possible. If we want to keep an attribute reachable only to check its value, then a `@property` is the way to do it.

```python
    self._language = language
    self._interactive = intercative

@property
def language(self):
    return self._language

@property
def interative(self):
    return self._interactive
```